thrift/src/main/thrift/atoms/storyquestions.thrift

```diff
@@ -32,4 +32,5 @@ struct StoryQuestionsAtom {
   4: optional list<QuestionSet> editorialQuestions
   5: optional list<QuestionSet> userQuestions
   6: optional shared.NotificationProviders notifications
+  7: optional bool closed = false
```

Comment:
In fact, there's already a `flags` object it can go in - https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/contentatom.thrift#L73